notebooks/5094_harmonisation.py

```diff
@@ -217,7 +218,10 @@
     user_overrides_gridding = user_overrides_gridding[user_overrides_gridding != "nan"]
 
 if model.startswith("REMIND"):
-    # advised on 5 June 2025 by Elmar
+    # READING form csv file located in "./data/interim/harmonisation_overrides"
```

Comment:
```suggestion
    # READING from csv file located in "./data/interim/harmonisation_overrides"
```

Move it to `data/raw/harmonisation_overrides` (raw is data we receive from somewhere else, interim is data we create but isn't the final product)

Can you check the overrides into the repo too please (`git add -f path/to/override.csv`)?